cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -0,0 +1,213 @@
+  - Members who are also members of other organizations must be re-added to your organization. If they sign in using an authentication method with [autoprovisioning](#autoprovisioning) enabled, they are automatically added upon successful sign-in. Otherwise, they must be re-invited. If they previously had the organizational admin role, it must be granted to them again.
+
+  During enablement of the feature, a list of affected members is shown, and those members are also notified individually.
+
+## Enable Cloud Organization SSO
+
+To enable Cloud Organization SSO:
+
+1. Log in to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) as an organizational admin.
+1. Go to **Organization** > **Authentication**.
+1. Next to **Enable Authentication**, click **Enable**.
+1. In the dialog, configure the custom URL your users will use to sign in. This value must be unique across {{ site.data.products.db }}. For more details, refer to [Update the custom URL](#update-the-custom-url).
+
+      {{site.data.alerts.callout_info}}
+      If you previously updated the custom URL from **Organization** > **Authentication** but did not enable Cloud Organization SSO, you are not prompted to configure it again. The value you have already configured is automatically used during enablement. To edit the custom URL again, click **Back** in the dialog.
+      {{site.data.alerts.end}}
+
+      Click **Next**
+
+1. The list of default authentication methods displays. By default, **Password**, **GitHub**, **Google**, and **Microsoft** are enabled.
+
+     To configure an authentication method, click its name and fill in the details. You can also do this later. Refer to [Configure advanced settings](#configure-advanced-settings).
+     To disable an authentication method, click **Disable**. You can also do this later.
+
+      {{site.data.alerts.callout_danger}}
+      If you disable password authentication, users can no longer authenticate to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) using the passwords that were  previously stored for them, and those passwords are not retained. Cockroach Labs recommends that you thoroughly test each SSO authentication method that you enable before disabling password authentication.
+      {{site.data.alerts.end}}
+
+     Click **Next**.
+
+1. Select **Confirm**, then click **Enable**.
+
+{{ site.data.products.db }} Organization SSO is enabled, and members can sign in by using the custom URL and selecting any enabled authentication method.
+
+## Update the custom URL
+
+After you enable Cloud Organization SSO, your users sign in use a custom URL that exposes only the authentication methods that you enable, rather than the public sign-in URL at `https://cockroachlabs.cloud`.
```

Comment:
```suggestion
After you enable Cloud Organization SSO, your users sign in at a custom URL that exposes only the authentication methods that you enable, rather than the public sign-in URL at `https://cockroachlabs.cloud`.
```